rgw/v2/tests/s3_swift/configs/test_object_lock_no_overwrite.yaml

```diff
@@ -0,0 +1,8 @@
+# upload type: non multipart
+# Related BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1890843
+# script: test_object_lock_no_overwrite.py - Test object lock configuration for bucket
```

Comment:
Please include polarian Id aswell

Reply:
@ckulal I have added code for BZ 2252336 as well , request you to review